### PR TITLE
Test

### PR DESCRIPTION
**Pull request by hotafrika**
1 pr

### CODE REVIEW

**Comment by hotafrika** [comment]
1 pr

File: dao/src/main/java/ds/DataSource.java [hunk 1]
```diff
@@ -53,6 +53,12 @@ public Connection getConnection() throws SQLException {
         return this.cpds.getConnection();
     }
 
-    public void closeConnection()
+    public void closeConnection(){
+        try {
+            this.getConnection().close();
+        } catch (SQLException e) {
```
Comment (karuneshapk) [inline dao/src/main/java/ds/DataSource.java]: че за херня

File: pom.xml [hunk 2]
```diff
@@ -25,15 +25,11 @@
             <artifactId>junit</artifactId>
             <version>4.12</version>
         </dependency>
-        <dependency>
-            <groupId>mysql</groupId>
-            <artifactId>mysql-connector-java</artifactId>
-            <version>5.1.38</version>
-        </dependency>
```
Comment (ipahomov) [inline pom.xml]: what is this???

File: pom.xml [hunk 3]
```diff
@@ -25,15 +25,11 @@
...
+       
         <dependency>
             <groupId>jstl</groupId>
             <artifactId>jstl</artifactId>
             <version>1.2</version>
         </dependency>
     </dependencies>
-</project>
```
Comment (ipahomov) [inline pom.xml]: del </project>

**Pull request merged**